irida_import/tests/integration/bash_scripts/install.sh

```diff
@@ -51,10 +49,10 @@ sed -i 's/#allow_path_paste: false/allow_path_paste: true/' galaxy.yml
 sed -i 's/#library_import_dir.*/library_import_dir: \//'  galaxy.yml
 
 # use MySQL instead of sqlite; to be configured to use a database user and name specified in README.md
-echo "  database_connection: postgres://galaxy:nicebigpasswordarebest@postgres/galaxy_test" | cat >> galaxy.yml
+echo "  database_connection: postgresql:///galaxy_test" | cat >> galaxy.yml
```

Comment:
Test are using two users for doing postgres setup. 
Postgres user for confirmation of the database and the running user of the CI testing. 
Suggestion would be to change to a single user for all postgresql database manipulation.

Reply:
Thanks. Changed to Run postgres command as regular user in Travis tests 5e3b56e4febc5067a0ba4ae8c795af5ee8c81f7d